src/vm/proftoeeinterfaceimpl.cpp

```diff
@@ -4110,6 +4110,14 @@ DWORD ProfToEEInterfaceImpl::GetModuleFlags(Module * pModule)
 
         // Intentionally not checking for flat, since NGEN PEImages never have flat
         // layouts.
+
+#ifdef FEATURE_READYTORUN
+        PEImageLayout * pPEImageLayout = pPEFile->GetLoaded();
+        if (pPEImageLayout != nullptr && pPEImageLayout->HasReadyToRunHeader())
```

Comment:
R2R images return false for HasNativeImage(), so this code path would never be taken.